server/monitor.go

```diff
@@ -2641,11 +2642,14 @@ func (s *Server) Jsz(opts *JSzOptions) (*JSInfo, error) {
 	}
 	s.js.mu.RUnlock()
 
-	if mg := s.js.getMetaGroup(); mg != nil {
-		if ci := s.raftNodeToClusterInfo(mg); ci != nil {
-			jsi.Meta = &MetaClusterInfo{Name: ci.Name, Leader: ci.Leader, Replicas: ci.Replicas, Size: mg.ClusterSize()}
+	if isLeader {
```

Comment:
I am a bit concerned here that if we just not set it like here users will get nil and code that was working before - they assumed cluster info was always present - would now encounter nil pointer exceptions.

So wdyt? I can make this just `jsi.Meta=&MetaClusterInfo{Leader: ci.Leader}` in that case - where we have a cluster but we arent a leader.  I think this will lead to fewer breakages as a result of this change, seems a better option than what we first thought.

Reply:
allright, I did that